website/mails/mails.py

```diff
@@ -251,6 +251,8 @@ def get_english_article(word):
 REQUEST_EXPORT = Mail('support_request', subject='[via OSF] Export Request')
 REQUEST_DEACTIVATION = Mail('support_request', subject='[via OSF] Deactivation Request')
```

Comment:
Based on this, it looks like the email sent to support if user the requests account deletion but has associated resources mentions the Deactivations Request in the subject and provides basic user information (id, profile url, email) via the support_request template. I don't know enough about how support handles these emails to comment on how account deletion is handled after they are contacted.

Reply:
@UdayVarkhedkar  I checked with Product and this is the expected format.